libs/compute_api/src/responses.rs

```diff
@@ -75,6 +75,7 @@ pub struct ComputeMetrics {
     pub start_postgres_ms: u64,
     pub config_ms: u64,
     pub total_startup_ms: u64,
+    pub load_libraries_ms: u64,
```

Comment:
Should we also track "number of libraries downloaded" or "size of largest library downloaded", or whatever we think is the best predictor for large `load_libraries_ms` time? It would help us distinguish slow starts that are ok from slow starts that are bugs

Reply:
"number of libraries downloaded" or "size of largest library downloaded"
I think both of these are good ideas

Reply:
maybe also "total number of megabytes downloaded"

Reply:
Doesn't hurt to do all 3 if it's easy. It's cheap to add stuff here. The values get logged at the end of startup, and they get sent to cplane, and cplane decides which values to pass to prometheus.

Reply:
done